catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -117,6 +124,17 @@ public ContentDirectoryMonitor(CamelContext camelContext, int maxRetries,
         this.maxRetries = maxRetries;
         this.delayBetweenRetries = delayBetweenRetries;
         this.configurationExecutor = configurationExecutor;
+        setBlacklist();
+    }
+
+    private void setBlacklist() {
+        String badFileProperty = System.getProperty("bad.files");
+        String badFileExtensionProperty = System.getProperty("bad.file.extensions");
+        badFiles = StringUtils.isNotEmpty(badFileProperty) ?
+                Arrays.asList(badFileProperty.split(",")) :
```

Comment:
I'm wondering if these split regex'es should include optional whitespace before and after the comma. (\s*,\s*).